views/home.ejs

```diff
@@ -5,14 +5,37 @@
 		<a href="/"><img src="/images/nba-logo.svg" alt="NBA logo" id="logoNBA" /></a>
     </div>
 
+
   <div class="container">
 	  <div class="container-fluid">
```

Comment:
![image](https://user-images.githubusercontent.com/41555434/157778108-8265bf2c-035b-426e-8f39-feac038b178a.png)

Maybe add something to indicate these are filters